.github/workflows/tests.yml

```diff
@@ -393,40 +482,38 @@ jobs:
 
     runs-on: ubuntu-latest
     steps:
-      - name: Download test results
-        uses: actions/download-artifact@v4
+      - name: Download Artifacts
+        uses: actions/download-artifact@v3
 
-      - name: Upload combined test results
-        # provides one downloadable archive of all .coverage/test-report.xml files
-        # of all matrix runs for further analysis.
+      - name: Upload Combined Test Results
+        # provides one downloadable archive of all matrix run test results for further analysis
         uses: actions/upload-artifact@v4
         with:
           name: test-results-${{ github.sha }}-all
-          path: test-results-${{ github.sha }}-*
-          retention-days: 90  # default: 90
+          path: test-results-*
```

Comment:
There's a warning now in the Summary:

> [aggregate](https://github.com/conda/conda/actions/runs/8058911732/job/22096776879#step:3:10)
No files were found with the provided path: test-results-*. No artifacts will be uploaded.

Reply:
I'm not seeing this in the latest run, can you confirm?

Reply:
Confirmed!